features/juniper-tca-collector/src/main/java/org/opennms/netmgt/collectd/tca/TcaCollectionHandler.java

```diff
@@ -102,28 +103,28 @@
 
 	private final RrdRepository m_repository;
 
-	private final ResourceStorageDao m_resourceStorageDao;
-
     private final ResourceType m_resourceType;
 
     private final LocationAwareSnmpClient m_locationAwareSnmpClient;
 
+	private final BlobStore m_blobStore;
+
 	/**
 	 * Instantiates a new TCA collection set.
 	 *
 	 * @param agent the agent
 	 * @param repository the repository
 	 */
-	public TcaCollectionHandler(SnmpCollectionAgent agent, RrdRepository repository, ResourceStorageDao resourceStorageDao,
-	        ResourceTypesDao resourceTypesDao, LocationAwareSnmpClient locationAwareSnmpClient) {
+	public TcaCollectionHandler(SnmpCollectionAgent agent, RrdRepository repository,
```

Comment:
Would be good to test the modified collectors at runtime, both on the core and on the Minion.

Just want to make sure that the dependencies get wired correctly, getting the collectors to actually collect data for these will be challanging.

Reply:
checking....

Reply:
I think we are good but I am not 100% sure. Here is what I did:

I debugged into TcaCollector and there the blobStore seems to be available:
![image](https://user-images.githubusercontent.com/1060134/124830056-fd907000-df3e-11eb-95ec-1d76a24056a8.png)

I didn't get to debug into Sftp3gppVTDXmlCollectionHandler but instead managed to debug into XmlCollector (same module) and was able to obtain the blobStore:
![image](https://user-images.githubusercontent.com/1060134/124830381-6a0b6f00-df3f-11eb-8768-8076f4e1ab51.png)

Reply:
Looks good - this means the new dependency is getting wired in. These collectors are rarely used, so this a low risk item.